client/components/CardGroup/ConnectedCardDumb.js

```diff
@@ -8,7 +8,9 @@ const CustomCard = ({ header, meta, description, history, link }) => {
       <Card.Content>
         {header && <Card.Header>{header}</Card.Header>}
         {meta && <Card.Meta>{meta}</Card.Meta>}
-        {description && <Card.Description>{description}</Card.Description>}
+        <Card.Description>
+          {description ? description : 'No description availabe.'}
```

Comment:
A small typo here just wanted to make you aware in case you have the time to fix it.